packages/commerce-sdk-react/src/auth/storage/base.ts

```diff
@@ -7,13 +7,15 @@
 
 export interface BaseStorageOptions {
     keySuffix?: string
+    keysExcludedFromSuffixing?: Array<string>
 }
 
 export abstract class BaseStorage {
     constructor(protected options: BaseStorageOptions = {}) {
         if (typeof this.options.keySuffix !== 'string') this.options.keySuffix = ''
     }
     protected getSuffixedKey(key: string): string {
+        if (this.options.keysExcludedFromSuffixing?.includes(key)) return key
```

Comment:
`dw_dnt` should not be site specific, so must not include the site suffix to this key